packages/keystone/KSv5v6/v5/prepareKeystone.js

```diff
@@ -185,6 +188,7 @@ function prepareKeystone ({ onConnect, extendKeystoneConfig, extendExpressApp, s
             new IpBlackListMiddleware(),
             new KeystoneTracingApp(),
             ...((apps) ? apps() : []),
+            ...(IS_ENABLE_DANGEROUS_GRAPHQL_PLAYGROUND ? [new GraphiqlApp({ graphiqlPath: '/admin/graphiql' })] : []),
```

Comment:
What about disable existing one via passing `IS_ENABLE_DANGEROUS_GRAPHQL_PLAYGROUND` to `playground` property a little lower in the code?

Reply:
for now I decided not to remove the old version - now there are links from different documentation and also the ability to change the path to the API

Reply:
disabled old playground